### DISCUSSION TITLE
I want to setup HA and DR

### DISCUSSION THREAD

**Discussion by dipakdotyadav** [quickwit-oss/quickwit#2833] [Q&A]
I am planning to use Quickwit for log ingestion (currently using Elasticsearch), I want to know if I can setup HA and DR

**Comment by guilload** [comment]
Hello @dipakdotyadav,

We will need more information to answer your question accurately. For example, are you planning on ingesting documents via a message queue (Kafka) or natively (HTTP ingest API)? Are you deploying Quickwit with Kubernetes? Which kind of storage do you target? S3, I assume?

Cheers

**Comment by fmassot** [comment]
@dipakdotyadav any update on your side?

**Comment by fmassot** [comment]
Closing the discussion @dipakdotyadav but you can reopening it if needed.
